src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -93,4 +102,25 @@ export class NavbarComponent implements OnInit {
     getImageUrl() {
         return this.isAuthenticated() ? this.accountService.getImageUrl() : null;
     }
+
+    /**
+     * Checks if the current component has a guided tour by comparing the current router url to manually defined urls
+     * that provide tours.
+     */
+    checkGuidedTourAvailability() {
+        if (this.router.url === '/overview') {
```

Comment:
I currently check if the page is an overview page and then display the 'start overview' menu item in the administration menu. In the future I have to change this since we cannot distinguish all pages only through the router.url